src/psyclone/psyGen.py

```diff
@@ -6070,8 +6142,41 @@ def __str__(self):
             ret = ret[:-2] + "]"  # Deletes last ", " and adds "]"
         else:
             ret += "Scalar"
+        if self.is_constant:
+            ret += ", constant, value={0}".format(self.constant_value)
         return ret + ">"
 
+    def copy(self):
+        '''Create and return a copy of this object.
```

Comment:
Add something like: "The new copy will not belong to a Symbol Table or Argument list even if the original object do."

Reply:
I think it is generically true that a copy of something will not have any references to it from other objects, so I don't think it makes sense to add this. Tin hat on.

Reply:
I am OK with this